pinot-common/src/main/java/org/apache/pinot/common/utils/FileUploadDownloadClient.java

```diff
@@ -77,1169 +77,1169 @@
  */
 @SuppressWarnings("unused")
 public class FileUploadDownloadClient implements AutoCloseable {
-  private static final Logger LOGGER = LoggerFactory.getLogger(FileUploadDownloadClient.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(FileUploadDownloadClient.class);
```

Comment:
The indentation is incorrect, which causes the whole file as changed

Reply:
Fixed.